modules/tinymce/src/plugins/accordion/test/ts/browser/QuirksTest.ts

```diff
@@ -0,0 +1,35 @@
+import { before, describe, it } from '@ephox/bedrock-client';
+import { PlatformDetection } from '@ephox/sand';
+import { TinyAssertions, TinyHooks, TinySelections } from '@ephox/wrap-mcagar';
+
+import Editor from 'tinymce/core/api/Editor';
+import Plugin from 'tinymce/plugins/accordion/Plugin';
+
+import * as AccordionUtils from '../module/AccordionUtils';
+
+describe('browser.tinymce.plugins.accordion.QuirksTest', () => {
+  const browser = PlatformDetection.detect().browser;
+  const hook = TinyHooks.bddSetup<Editor>(
+    {
+      plugins: 'accordion',
+      base_url: '/project/tinymce/js/tinymce',
+      indent: false
+    },
+    [ Plugin ]
+  );
+  before(function () {
+    if (!browser.isSafari()) {
+      this.skip();
+    }
```

Comment:
could it make sense to put this `if` in the assertion?
we need to be in a valid selection clicking on the expand arrow also for the other browsers

Reply:
done